### DISCUSSION TITLE
Enterprise Guardrails for Codespaces

### DISCUSSION THREAD

**Discussion by rishabhsaha** [cse-labs/kubernetes-in-codespaces#11] [Q&A]
Additional details around how codespace handles Networking, Security, Identity etc.

**Comment by bartr** [comment]
Identity is provided by your GitHub ID. A GitHub Team or GitHub Enterprise can determine who in that org has access to Codespaces and which repos they can open with Codespaces. Since a Codespaces is "just a dev box", you can clone / push to any GitHub repo that you have permissions to.

**Comment by bartr** [comment]
With one of the GitHub Enterprise SKUs, you can run your Codespaces servers in your Azure subscription which gives you more control over security.
